testing/functional_tests.cfg

```diff
@@ -3,32 +3,42 @@ test_dir = fates_allom_ftest
 test_exe = FATES_allom_exe
 out_file = allometry_out.nc
 use_param_file = True
+datm_file =
+datm_file_url =
```

Comment:
we shouldn't need to have these here, only for the classes that need them.

Reply:
Done with c65df334.